k8s/gvclient.go

```diff
@@ -32,26 +34,50 @@ type groupVersionClient struct {
 
 func (g *groupVersionClient) get(ctx context.Context, identifier resource.Identifier, plural string,
 	into resource.Object) error {
+	ctx, span := GetTracer().Start(ctx, "kubernetes-get")
+	defer span.End()
 	sc := 0
 	request := g.client.Get().Resource(plural).Name(identifier.Name)
 	if strings.TrimSpace(identifier.Namespace) != "" {
 		request = request.Namespace(identifier.Namespace)
 	}
 	bytes, err := request.Do(ctx).StatusCode(&sc).Raw()
+	span.SetAttributes(
+		attribute.Int("http.response.status_code", sc),
+		attribute.String("http.request.method", http.MethodGet),
+		attribute.String("server.address", request.URL().Hostname()),
+		attribute.String("server.port", request.URL().Port()),
```

Comment:
Do we need these? They are already in the URL and I don't think I personally would be searching / grouping by these.

Reply:
`server.address` is listed as required in the [Semantic Conventions for HTTP Spans](https://github.com/open-telemetry/semantic-conventions/blob/098db1ca510da01fe941f6c6308aad5000def910/docs/http/http-spans.md#http-client), and `server.port` is required if non-default for the protocol, which I didn't want to have a check each time for.

Reply:
Good to know. Thanks!